src/debug/debug_stream/Kconfig

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+config SOF_DEBUG_STREAM_SLOT
+	bool "Enable SOF debug stream debug window slot"
+	help
```

Comment:
isn't it "n" by default?

Reply:
Lets call it out though, we really should have a default 'n' or 'y' for all Kconfig entries.

Reply:
The Zephyr coding style recommends to avoid the redundant `default n` for a couple reasons:

https://docs.zephyrproject.org/latest/build/kconfig/tips.html#redundant-defaults

`git -C zephyr/ grep -h default.n -- '*Kconfig' | wc -l` is 10 times smaller than `default y` so it seems mostly followed in Zephyr.

The same SOF stats look very different indeed.